brabbuss/overlook
# Homepage/initial build

### What does this PR do?

655df63 add user lookup for manager dashboard
a1e415c add my account to user dash, where user can see past bookings
629d9eb add ability for user to book an available room
d54bcfd update methods to allow dynamic update of available rooms after user books room

### What are the relevant tickets (please link)?
#19 
#17 
#14 
#18 

### How should this be tested?
1. should be able to book a room and see it in the user history in the console, should see available rooms update (the room just booked should disappear)

### New Questions/Thoughts?:
1. dynamic update logic should be applied to the user stats/manager stats as well
